scripts/combine/setupCombine.py

```diff
@@ -235,7 +235,7 @@ def setup(args, inputFile, inputBaseName, inputLumiScale, fitvar, genvar=None, x
 
     if isPoiAsNoi:
         constrainMass = False if isTheoryAgnostic else True
-        poi_axes = datagroups.gen_axes_names if genvar is None else genvar
+        poi_axes = datagroups.gen_axes_names if args.genAxes is None else args.genAxes
```

Comment:
I think this change should be reverted in order that the poiAsNoi works for multiple input files with different gen axes

Reply:
ok!

Reply:
it doesn't work with the theory agnostic out of the box, trying to understand why